mne/cov.py

```diff
@@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
```

Comment:
No ... only within local functions, perhaps because the default dict is not always the same?  For example, in <code>prepare_noise_cov</code>, starting at line 1128: 

    scalings_ = dict(mag=1e12, grad=1e11, eeg=1e5)
    if isinstance(scalings, dict):
        scalings_.update(scalings)